src/sensors/ADCResistanceSensor.h

```diff
@@ -0,0 +1,70 @@
+/*
+	SlimeVR Code is placed under the MIT license
+	Copyright (c) 2024 Eiren Rain & SlimeVR contributors
+
+	Permission is hereby granted, free of charge, to any person obtaining a copy
+	of this software and associated documentation files (the "Software"), to deal
+	in the Software without restriction, including without limitation the rights
+	to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+	copies of the Software, and to permit persons to whom the Software is
+	furnished to do so, subject to the following conditions:
+
+	The above copyright notice and this permission notice shall be included in
+	all copies or substantial portions of the Software.
+
+	THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+	IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+	FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+	AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+	LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+	OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+	THE SOFTWARE.
+*/
+#ifndef ADC_FLEX_SENSOR_H
+#define ADC_FLEX_SENSOR_H
+
+#include "sensor.h"
+#include "sensorinterface/SensorInterface.h"
+
+class ADCResistanceSensor : Sensor {
+public:
+	static constexpr auto TypeID = ImuID::ADC_RESISTANCE;
+
+	ADCResistanceSensor(
+		uint8_t id,
+		uint8_t pin,
+		float VCC,
+		float resistanceDivider,
+		float smoothFactor = 0.1f
+	)
+		: Sensor(
+			"ADCResistanceSensor",
+			ImuID::ADC_RESISTANCE,
+			id,
+			pin,
+			0.0f,
+			new SlimeVR::EmptySensorInterface
```

Comment:
Instead of new-ing it, it would be better to have a single global instance of the empty sensor interface and use that

Reply:
doesn't matter, it happens once and it's empty. instead of bothering with scope it's easier to do that, imo.